CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ All notable changes to this project will be documented in this file.
 
 - Add `context.redirect(path)` method to be used from inside route actions (PLANNED)
 
+### [v1.3.0] - [unreleased]
+
+- Handle `null` same way as `undefined`
+- Return `null` instead of `undefined` to signal no match. (BREAKING CHANGE)
```

Comment:
@frenzzy It's not so breaking.. API remains same but behavior is slightly modified. What you mean by follow semver? I think shift up minor in this case is fine.

@koistya What you think?